.env.example

```diff
@@ -1,14 +1,14 @@
-# L1 NODE
-REACT_APP_L1_RPC_URL=http://localhost:8545
+# ETHEREUM
+REACT_APP_ETHEREUM_RPC_URL=http://localhost:8545
+REACT_APP_ETHEREUM_BRIDGE_CONTRACT_ADDRESS=0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9
 
-# L2 NODE
-REACT_APP_L2_RPC_URL=http://localhost:8123
+# POLYGON_HERMEZ
+REACT_APP_POLYGON_HERMEZ_RPC_URL=http://localhost:8123
+REACT_APP_POLYGON_HERMEZ_BRIDGE_CONTRACT_ADDRESS=0x9d98deabc42dd696deb9e40b4f1cab7ddbf55988
+REACT_APP_POLYGON_HERMEZ_NETWORK_ID=1
 
 # BRIDGE
```

Comment:
I would rename this comment to `BRIDGE API`

Reply:
Done